org.audacityteam.Audacity.appdata.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
In the end any appdata improvements should go upstream.

Reply:
Of course, I will that. I just wanted to get the update out.